src/benchmark/options.ts

```diff
@@ -2,9 +2,9 @@ import {AverageCalculationEnum, BenchmarkOpts, ConvergenceEnum} from "../types.j
 
 export const defaultBenchmarkOptions: Required<BenchmarkOpts> = {
   minRuns: 1,
-  maxRuns: Infinity,
+  maxRuns: Number.POSITIVE_INFINITY,
```

Comment:
We should delete this rule so its the same as `ssz` repo to keep everything consistent